### PR TITLE
8916 contracts table component

### PR DESCRIPTION
**Pull request by brianpetway**
https://federal-spending-transparency.atlassian.net/browse/DA-8916

### CODE REVIEW

File: src/components/contracts-table/contracts-table.module.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+.heading {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.icon {
+  font-size: 16px;
+  color: red;
```
Comment (anjenkin) [inline src/components/contracts-table/contracts-table.module.scss:17]: this font color can be removed

File: src/components/contracts-table/contracts-table.module.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  font-weight: $body-copy-weight;
+  color: $font-black;
+  padding-top: 0.2rem;
+}
+
+.amount {
+  fonst-size: 16px;
+  font-weight: $bold;
+  color: $rdMdBlue;
+  //text-decoration: underline;
```
Comment (anjenkin) [inline src/components/contracts-table/contracts-table.module.scss:50]: remove comment?

**Review by anjenkin — CHANGES_REQUESTED** [review]
A few things - 
* there shouldn't be a line under the recipient number, just the name and obligation
* per Zeplin, there's more space between the agency name and the line (about 11 px)
* the recipient name is lower case in the mocks (you'll need to ask Sabrina or Lindsay for an updated data file)
* See comments inline

**Comment by brianpetway** [comment]
Changes made. I will ask grace and kelli about the new csv without all caps in recipients.

**Review by anjenkin — APPROVED** [review]

**Pull request merged**